docs/src/components/blog/material-ui-v6-is-out/ThemeTokens.js

```diff
@@ -80,7 +82,7 @@ export default function TemplateCarousel() {
   const shape = ['--mui-shape-borderRadius'];
   const zIndex = Object.keys(defaultTheme.vars.zIndex).map((key) => `--mui-zIndex-${key}`);
   return (
-    <CssVarsProvider theme={defaultTheme}>
+    <ThemeProvider theme={() => defaultTheme}>
```

Comment:
Change to a function to ensure that it does not inherit the upper branding theme.
This demo is to show default tokens of Material UI theme.